public/index.html

```diff
@@ -4,20 +4,846 @@
     <meta charset="utf-8">
     <meta content='width=device-width, initial-scale=1.0, maximum-scale=5.0, user-scalable=0' name='viewport' />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <link rel="stylesheet" href="/css/leaflet.css" async/>
-    <link rel="stylesheet" href="/css/material_icons.css">
-    <script src="/js/recaptcha.js" async defer></script>
-    <script src="/js/charts.js" async defer></script>
+    <style>
```

Comment:
Why can't these CSS be kept inside the public folder?

Reply:
Actually when using webpack for development it can be told to use public folder for asset management. Since webpack does not honour the relative paths mentioned in the html file when it creates the dev server these hard coded paths would not work (unless the paths are adjusted according to where the server is running). Moreover the paths would then have to be changed according to dev and prod build as dev-server uses public folder and prod uses the build folder.